independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -362,11 +365,13 @@ public InputStream getResourceAsStream(String unsanitisedName) {
 
     @Override
     public Class<?> loadClass(String name) throws ClassNotFoundException {
+        classLoaderEventListeners.forEach(l -> l.loadClass(name, this.name));
```

Comment:
I think an enhanced for-loop would be better here for performance purposes?

Reply:
That's certainly true, but as this is the dev and test mode ClassLoader, it shouldn't make much of a difference.

Sanne knows I was real picky about the prod ClassLoader, but for this one I don't think we need to be that extreme :)

Reply:
I generally prefer avoiding lambdas myself, so I'd be happy to change this if you prefer.

The reason I went with it in this case is that this code only applies to test/dev mode - so I'm assuming the one-liner is less intrusive and maintenance friendly. No strong opinions, I just thought this stuff shouldn't be a distraction from the overall ClassLoading business :)

Reply:
If changing that makes the test or dev mode run faster, I'd say go for it 😉

Reply:
done